### PR TITLE
Handle PrimaryKey as str on Python3. Avoid recreating resized array. Fix reshaping division.

### PR DESCRIPTION
**Pull request by cielavenir**

### CODE REVIEW

**Comment by cielavenir** [comment]
In https://github.com/mujin/mujinwebstackclientpy/pull/22#issuecomment-1687557707 I said this pull request has been tested.

Is it possible to get this merged before orchestrator migration?

**Review by felixvd — APPROVED** [review]
I don't see anything wrong with this.

**Review by ziyan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Handle PrimaryKey as str on Python3
```
Comment (woswos) [inline CHANGELOG.md:1]: @cielavenir is there a reason why this change wasn't versioned?

cc: @felixvd
